cpp/include/kvikio/error.hpp

```diff
@@ -16,8 +16,6 @@
 #pragma once
 
 #include <cstring>
```

Comment:
Do we need `<stdexcept>` for `std::runtime_error`?

Reply:
Thanks. Done.